packaging/src/docker/conf/hive-site.xml

```diff
@@ -60,4 +60,20 @@
         <name>metastore.metastore.event.db.notification.api.auth</name>
         <value>false</value>
     </property>
+    <property>
+        <name>iceberg.catalog</name>
+        <value>rest</value>
+    </property>
+    <property>
+        <name>iceberg.catalog.rest.type</name>
+        <value>rest</value>
+    </property>
+    <property>
+        <name>iceberg.catalog.rest.uri</name>
+        <value>http://playground-gravitino:9001/iceberg</value>
```

Comment:
HMS Iceberg catalog would be merged soon, so i think it would be better to use that one as default

Reply:
Removed.